### ISSUE TITLE
Investigate flaky test in security dashboards functional test

### ISSUE THREAD

**Issue by DarshitChanpura** [opensearch-project/security-dashboards-plugin#1070]
During the latest functional test build for version 2.2, a test in permissions_spec.js: "should create new action group successfully by selecting `Create from blank`" intermittently fails due to cy.intercept() call in `mockPermissions` in [command.js](https://github.com/opensearch-project/opensearch-dashboards-functional-test/blob/63978b6597918056f9a5cd22427eaaf14a869996/cypress/utils/plugins/security/commands.js#L61)

Failure artifact:
[security.zip](https://github.com/opensearch-project/security-dashboards-plugin/files/9329327/security.zip)

**Comment by peternied** [comment]
@DarshitChanpura Shouldn't this bug be in the codebase where that test is maintained, see [permissions_spec.js](https://github.com/opensearch-project/opensearch-dashboards-functional-test/blob/main/cypress/integration/plugins/security/permissions_spec.js)?

**Comment by DarshitChanpura** [comment]
yes, that's true. We can transfer it to that repo but security team will be investigating it.
